src/chunkserver/hddspacemgr.cc

```diff
@@ -18,8 +18,13 @@
    along with SaunaFS  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include "common/platform.h"
+
 #include "hddspacemgr.h"
 
+#include "common/platform.h"
+#include "common/saunafs_error_codes.h"
+
```

Comment:
common/platform.h is repeated.
common/saunafs_error_codes.h must be in the proper group (below).